packages/ui/src/select/index.tsx

```diff
@@ -0,0 +1,45 @@
+'use client'
+
+import { HTMLAttributes } from 'react';
+import CustomSelect from 'react-select';
+import styles from './select.css';
+import clsx from 'clsx';
+import customStyles from './customStyles';
+
+export default function Select({ label, options, ...props }: SelectProps) {
+  const handleChange = (selectedOption: optionObject | null) => {
+    // TODO: 선택된 옵션을 받아서 처리.
+    console.log(`Option selected:`, selectedOption);
+  };
+  
+  return (
+    <div className={clsx(styles.container, props.className)}>
+      <label className={styles.label}>
+        {label}
+      </label>
+      <CustomSelect
+        options={options}
+        className={styles.select}
+        styles={customStyles}
+        isSearchable={false}
+        placeholder={props.placeholder}
+        onChange={handleChange}
+        components={{
+          IndicatorSeparator: () => null,
+        }}
+      />
+    </div>
+  );
+}
+
+interface optionObject { 
+  value: string,
+  label: string 
+};
```

Comment:
`interface`나 `type`은 파스칼 케이스로 작성하는 것이 일반적이라고 알고있습니다!
작성 해주신 다른 `interface`들과 마찬가지로 파스칼 케이스로 작성하는 것은 어떨까요?
```suggestion
interface OptionObject { 
  value: string,
  label: string 
};
```

Reply:
`Object`라는 이름대신 이 친구가 어떤 인터페이스인지 더 명확하게 나타내는 이름을 쓰는 것도 좋을 것 같습니다..!
```suggestion
interface Option { 
  value: string,
  label: string 
};
```
혹은 

```suggestion
interface OptionType { 
  value: string,
  label: string 
};
```
같은 이름은 어떠세요?

Reply:
린트가 없다보니 저도 모르게 습관적으로 카멜케이스로 작성했네요 ㅠ 짚어주셔서 감사합니다. 제안해주신 식별자명으로 수정해볼게요